### ISSUE TITLE
[Accessibility] Slider: Doesn't Announce Current Value

### ISSUE THREAD

**Issue by HalcyonPride** [OfficeDev/office-ui-fabric-react#7993]
### Environment Information 
 
- __Package version(s)__: Latest
- __Browser and OS versions__: Edge; Windows 10 Enterprise, version 1809

### Please provide a reproduction of the bug in a codepen:
https://developer.microsoft.com/en-us/fabric#/components/slider

 
#### Actual behavior:
 
Narrator doesn't announce the current value as you move the slider.
 
#### Expected behavior:
 
Narrator should announce the current value.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: High 
 
Products/sites affected: ODB

**Comment by JasonGore** [comment]
The same behavior also occurs with an [ARIA slider example](https://www.w3.org/TR/wai-aria-practices/examples/slider/slider-1.html) indicating that this is an issue with either Edge or Narrator. I've also tested with NVDA, which announces the Slider value in Chrome, but not in Edge. As a result, I suspect this is an issue with Edge.

I'm working to create an issue with the Edge team now.

**Comment by JasonGore** [comment]
Further investigation shows that [this ARIA slider](https://www.w3.org/TR/wai-aria-practices/examples/slider/slider-2.html) example works correctly. The primary difference appears to be the presence of the `aria-valuetext` value. when this value is missing, MS Edge/Narrator do not appear to fall back to `aria-valuenow` attribute for narration.

`aria-valuetext` is not specifically required by ARIA standards for narration, so this is still considered an issue with MS Edge & Narrator. Consumers of `Slider` could add the `aria-valuetext` attribute as a temporary workaround.

**Comment by HalcyonPride** [comment]
@JasonGore, I concur with your assessment. Since you're gonna open an issue with Edge, I'll close this Bug.
